index.script.js

```diff
@@ -0,0 +1,39 @@
+function add() {
+    let field1=document.getElementById('num1').value ;
```

Comment:
can move element searches outside the functions and only use it one. why do `document.getElementById('num1')` each time in each function?

Move it out and save in variable one.

then in side function do this:  `let field1 = variable.value`